src/System.Runtime/tests/System/ActivatorTests.cs

```diff
@@ -392,5 +395,159 @@ static void TestingActivationAttributes1()
+            Assert.NotNull(oh);
+            Assert.Equal(returnedFullNameType, oh.Unwrap().GetType().FullName);
+        }
+
+        public static IEnumerable<object[]> TestingCreateInstanceObjectHandleFullSignature()
+        {
+            //string assemblyName, string typeName, bool ignoreCase, BindingFlags bindingAttr, Binder binder, object[] args, CultureInfo culture, object[] activationAttributes, returnedFullNameType
+            yield return new object[] { "TestLoadAssembly", "PublicClassSample", false, BindingFlags.Public | BindingFlags.Instance, Type.DefaultBinder, new object[0], CultureInfo.InvariantCulture, null, "PublicClassSample" };
+            yield return new object[] { "testloadassembly", "publicclasssample", true, BindingFlags.Public | BindingFlags.Instance, Type.DefaultBinder, new object[0], CultureInfo.InvariantCulture, null, "PublicClassSample" };
+            yield return new object[] { "TestLoadAssembly", "PublicClassSample", false, BindingFlags.Public | BindingFlags.Instance, Type.DefaultBinder, new object[1] { 1 }, CultureInfo.InvariantCulture, null, "PublicClassSample" };
+            yield return new object[] { "testloadassembly", "publicclasssample", true, BindingFlags.Public | BindingFlags.Instance, Type.DefaultBinder, new object[1] { 1 }, CultureInfo.InvariantCulture, null, "PublicClassSample" };
+
+            yield return new object[] { "TestLoadAssembly", "PrivateClassSample", false, BindingFlags.NonPublic | BindingFlags.Public | BindingFlags.Instance, Type.DefaultBinder, new object[0], CultureInfo.InvariantCulture, null, "PrivateClassSample" };
+            yield return new object[] { "testloadassembly", "privateclasssample", true, BindingFlags.NonPublic | BindingFlags.Public | BindingFlags.Instance, Type.DefaultBinder, new object[0], CultureInfo.InvariantCulture, null, "PrivateClassSample" };
+            yield return new object[] { "TestLoadAssembly", "PrivateClassSample", false, BindingFlags.NonPublic | BindingFlags.Public | BindingFlags.Instance, Type.DefaultBinder, new object[1] { 1 }, CultureInfo.InvariantCulture, null, "PrivateClassSample" };
+            yield return new object[] { "testloadassembly", "privateclasssample", true, BindingFlags.NonPublic | BindingFlags.Public | BindingFlags.Instance, Type.DefaultBinder, new object[1] { 1 }, CultureInfo.InvariantCulture, null, "PrivateClassSample" };
+
+            yield return new object[] { null, typeof(PublicType).FullName, false, BindingFlags.Public | BindingFlags.Instance, Type.DefaultBinder, new object[0], CultureInfo.InvariantCulture, null, typeof(PublicType).FullName };
+            yield return new object[] { null, typeof(PrivateType).FullName, false, BindingFlags.NonPublic | BindingFlags.Public | BindingFlags.Instance, Type.DefaultBinder, new object[0], CultureInfo.InvariantCulture, null, typeof(PrivateType).FullName };
+        }
+        
+        [Fact]        
+        static void CreateInstanceAssemblyResolve()
```

Comment:
@jkotas with this test i wanted to cover https://github.com/dotnet/coreclr/blob/master/src/System.Private.CoreLib/src/System/Activator.cs#L132 but after coverage analisys seems it doesn't pass here, any thoughts?

Reply:
I do not see anything obvious. It is possible that the `assemblyFromResolveEvent` path is unreachable code in CoreCLR if this does not hit it.

Reply:
What do you think is bettere here?Do we need to go down coreclr bit to understand why path is unreachable?Or this tests is enough for the scope of CreateInstance?

Reply:
I think you need to use invalid assembly name, like `,,,,` to trigger the assembly resolve code path. AssemblyResolve event gets called for invalid names and can fix them up to something valid. For example, consider following program:

```csharp
using System;
using System.Reflection;

class Program
{
    static void Main(string[] args)
    {
        AppDomain.CurrentDomain.AssemblyResolve += CurrentDomain_AssemblyResolve;
        Console.WriteLine(Assembly.Load(",,,,"));
    }

    private static System.Reflection.Assembly CurrentDomain_AssemblyResolve(object sender, ResolveEventArgs args)
    {
        Console.WriteLine("AssemblyResolve: " + args.Name);
        return typeof(Program).Assembly;
    }
}
```

It will print:
```
AssemblyResolve: ,,,,
test, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null
```

Reply:
I'll try and let you know! Thank's for tip!

Reply:
@jkotas it works(as expected :smirk:)!Branch covered...the only uncovered branch is https://github.com/dotnet/coreclr/blob/master/src/System.Private.CoreLib/src/System/Activator.cs#L140 `AssemblyContentType.WindowsRuntime` is there a way to test WinRT assembly?

Reply:
`Type.GetType("Windows.Foundation.Collections.StringMap, Windows, Version=255.255.255.255, Culture=neutral, PublicKeyToken=null, ContentType=WindowsRuntime")` returns WinRT type on .NET Core 2.1 today.

So I would expect that using `Windows.Foundation.Collections.StringMap` as type name and `Windows, Version=255.255.255.255, Culture=neutral, PublicKeyToken=null, ContentType=WindowsRuntime` as assembly name here should hit this path.

For `PlatformDetection.IsNotWinRTSupported` platforms, it should throw exception.

Reply:
Ok WinRT branch covered.
The last two branch i can't trigger are:
https://github.com/dotnet/coreclr/blob/master/src/System.Private.CoreLib/src/System/Activator.cs#L154 i cannot return null from `assemblyFromResolveEvent` or `assemblyString == null` because throw exception.
https://github.com/dotnet/coreclr/blob/master/src/System.Private.CoreLib/src/System/Activator.cs#L167 i cannot trigger return null branch...maybe with some `bindingAttr` config...but i did't find anything so far.

Reply:
> i cannot trigger return null branch

Agree. This branch looks like unreachable code. I think it should be fine to delete it.

Reply:
Ok all branch of main method covered!